locales/zh_CN/LC_MESSAGES/gui.po

```diff
@@ -3,89 +3,91 @@
 # This file is distributed under the same license as the taichi package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
 #
-#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: taichi 0.6.2\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-05-07 01:24-0400\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2020-05-14 10:20+0800\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.8.0\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: zh_CN\n"
+"X-Generator: Poedit 2.3.1\n"
 
 #: ../../gui.rst:4
 msgid "GUI system"
-msgstr ""
+msgstr "GUI系统"
 
 #: ../../gui.rst:6
 msgid ""
-"Taichi has a built-in GUI system to help users display graphic results "
-"easier."
-msgstr ""
+"Taichi has a built-in GUI system to help users display graphic results easier."
+msgstr "Taichi具有内置的GUI系统，可帮助用户更轻松地显示图形结果。"
 
 #: ../../gui.rst:10
 msgid "Create a window"
-msgstr ""
+msgstr "创建一个窗口"
 
 #: ../../gui.rst
 msgid "Parameters"
-msgstr ""
+msgstr "参量"
 
 #: ../../gui.rst:15
 msgid "(string) the window title"
-msgstr ""
+msgstr "（字符串）窗口标题"
 
 #: ../../gui.rst:16
 msgid "(scalar or tuple) resolution / size of the window"
-msgstr ""
+msgstr "（标量或元组）分辨率/窗口大小"
 
 #: ../../gui.rst:17
 msgid "(optional, RGB hex) background color of the window"
-msgstr ""
+msgstr "（可选，RGB十六进制）窗口的背景颜色"
 
 #: ../../gui.rst
 msgid "Returns"
-msgstr ""
+msgstr "返回"
 
 #: ../../gui.rst:18
 msgid "(GUI) an object represents the window"
-msgstr ""
+msgstr "(GUI)对象代表窗口"
```

Comment:
Rule update: for those `API Reference` styled bracket, do not use English bracket even if the content is fully English, this helps keep style unified.
```suggestion
msgstr "（GUI）对象代表窗口"
```